department-of-veterans-affairs/va.gov-cms
# VACMS-3291: CLP Spotlight Link Teaser Validation

… 3 to match config

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/3921

## Testing done


## Screenshots


## QA steps

- Login to preview environment
- Navigate here: /node/add/campaign_landing_page
- Fill out all required fields, enable Spotlight Panel
- Add 3 spotlight link teasers
- [ ] Confirm you are able to save successfully

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
